tidb-cloud/integrate-tidbcloud-with-n8n.md

```diff
@@ -0,0 +1,294 @@
+
+</div>
+<div label="Insert">
+
+- **Credential for TiDB Cloud API**: Only supports TiDB Cloud API key authentication. Refer to [Get TiDB Cloud API Key](#prerequisites-get-tidb-cloud-api-key).
+- **Project**: The TiDB Cloud project name.
+- **Operation**: The operation of this node. Refer to [Support Operation](#support-operation).
+- **Cluster**: The TiDB Cloud cluster name. In the Create Serverless Cluster operation, you need to enter one name for your new cluster . While in other operations, you should choose one existing cluster.
+- **Password**: The password of TiDB Cloud cluster.
+- **User**: The username of your TiDB Cloud cluster.
+- **Database**: The database name.
+- **Table**: The table name. You can use `From list` mode to choose one or `Name` mode to type table name manually.
+- **Columns**: The comma-separated list of the input item's property which should used as columns for the new rows. Item is the data sent from one node to another. A node performs its action on each item of incoming data. For more information about item in n8n, see [n8n documentation](https://docs.n8n.io/workflows/items/).
+
+</div>
+<div label="Update">
+
+- **Credential for TiDB Cloud API**: Only supports TiDB Cloud API key authentication. Refer to [Get TiDB Cloud API Key](#prerequisites-get-tidb-cloud-api-key).
+- **Project**: The TiDB Cloud project name.
+- **Operation**: The operation of this node. Refer to [Support Operation](#support-operation).
+- **Cluster**: The TiDB Cloud cluster name. In the Create Serverless Cluster operation, you need to enter one name for your new cluster. While in other operations, you should choose one existing cluster.
+- **Password**: The password of TiDB Cloud cluster.
+- **User**: The username of your TiDB Cloud cluster.
+- **Database**: The database name.
+- **Table**: The table name. You can use `From list` mode to choose one or `Name` mode to type table manually.
+- **Update Key**: The name of the item's property which decides which rows in the database should be updated. Item is the data sent from one node to another. A node performs its action on each item of incoming data. For more information about item in n8n, see [n8n documentation](https://docs.n8n.io/workflows/items/).
+- **Columns**: The comma-separated list of the input item which should used as columns for the rows to update.
+
+</div>
+</SimpleTab>
+
+### Limitations 
+ 
+1. Normally only one SQL statement is allowed in the **Execute SQL** operation. If you want to execute more than one statement in a single operation, you need to manually enable [`tidb_multi_statement_mode`](https://docs.pingcap.com/tidb/dev/system-variables#tidb_multi_statement_mode-new-in-v4011).
```

Comment:
Use the link in *TiDB Cloud documentation* rather than *TiDB documentation*.
```suggestion
- Normally only one SQL statement is allowed in the **Execute SQL** operation. If you want to execute more than one statement in a single operation, you need to manually enable [`tidb_multi_statement_mode`](https://docs.pingcap.com/tidbcloud/system-variables#tidb_multi_statement_mode-new-in-v4011).
```